tests/system/tests/bash/count_assets_https.bash

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+curl -sk https://localhost:1995/foglamp/asset | jq -S '.'
```

Comment:
now we should use http and hence `-k`

recommendation: when https intstead of -k, `--insecure` adds more readability

Reply:
This is the https script, so it should use https: and -k.
-k is commonly used, --insecure has been introduced later and pretty unknown. I could say that it would add more uncertainty and developers would have to check on man what that means (as I did, because I had no idea it is the same of -k).